package.json

```diff
@@ -25,7 +25,7 @@
     "resin-register-device": "^2.0.0",
     "request-progress": "^0.3.1",
     "rwlock": "^5.0.0",
-    "sqlite3": "~3.0.4",
+    "sqlite3": "3.0.9",
```

Comment:
So `3.0.10`/`3.1.0` remove the fix?  Because both `~3.0.9` or `^3.0.9` will force `3.0.9` as the minimum version